crates/cxx-qt-lib-headers/include/gui/qguiapplication.h

```diff
@@ -18,5 +19,11 @@ namespace cxxqtlib1 {
 ::std::unique_ptr<QGuiApplication>
 qguiapplicationNew(const QVector<QByteArray>& args);
 
+void
+qapplicationSetFont(QGuiApplication& app, const QFont& font);
```

Comment:
these should probably be called `qguiapplication` rather than `qapplication` (sorry)